src/redux/configueStore.js

```diff
@@ -0,0 +1,9 @@
+import { createStore, combineReducers } from 'redux';
+import bookReducer from './books/books';
+import statusReducer from './categories/categories';
+
+const rootReducer = combineReducers({ bookReducer, statusReducer });
```

Comment:
- [ ] Please give a key to your reducers for example
- books: bookReducer
- status: statusReducer